pkg/logs/schedulers/ad/scheduler.go

```diff
@@ -210,6 +210,13 @@ func (s *Scheduler) createSources(config integration.Config) ([]*sourcesPkg.LogS
 		}
 
 		source := sourcesPkg.NewLogSource(configName, cfg)
+		if source.Config.IntegrationName == "" {
+			// If the log source comes from a config file it does not specify a log source
+			// We try to match it with the config name that is most likely the integration name
+			// If it comes from a container environment, the name was computed based on the `check_names`
+			// Running next to it.
```

Comment:
What is running next to what?

Reply:
Rephrased it, let me know what you think